core/src/main/java/org/elasticsearch/bootstrap/BootstrapInfo.java

```diff
@@ -120,4 +120,8 @@ public Object remove(Object key) {
         }
         return SYSTEM_PROPERTIES;
     }
+
+    public static void init() {
```

Comment:
> It may even before useful to just call it more properly something like initBeforeSecurityManager.

I think that `init` is fine and the name shouldn't be tied to implementation details about the rest of our system. A comment at the call site like you suggested is the right thing.